ClubManagement/ClubManagement/Activities/MemberLocationActivity.cs

```diff
@@ -71,46 +71,46 @@ protected override void OnCreate(Bundle savedInstanceState)
 
             Cheeseknife.Inject(this);
 
-            FragmentManager.BeginTransaction()
-                           .Replace(Resource.Id.memberFrament, personGoTimesFragment)
-                .Commit();
+            //FragmentManager.BeginTransaction()
```

Comment:
Please remove the code instead of disable them

Reply:
Done